frontend/pages/club/[club]/index.tsx

```diff
@@ -3,7 +3,7 @@ import { DesktopActions, MobileActions } from 'components/ClubPage/Actions'
 import AdvisorList from 'components/ClubPage/AdvisorList'
 import ClubApprovalDialog from 'components/ClubPage/ClubApprovalDialog'
 import Description from 'components/ClubPage/Description'
-import Events from 'components/ClubPage/Events'
+// import Events from 'components/ClubPage/Events'
```

Comment:
You can probably go ahead and delete any commented-out code, deferring to @julianweng on this

Reply:
Probably a good idea, nice catch